src/components/swap/SwapBox.vue

```diff
@@ -183,6 +209,154 @@ export default defineComponent({
     toggleshowMaxTooltip() {
       this.showMaxTooltip = !this.showMaxTooltip;
     },
+    async onSwap() {
+      if (!this.swapFrom.value) {
+        console.error("You need to estimate a swap amount!");
```

Comment:
if you rebase from main, you will be able to do 
`throw new UserNotificationError(..., ..., ...)`
and that will automatically show the error toast.